ui/src/views/WorkerManager/WMViewWorkerPools/index.jsx

```diff
@@ -146,6 +147,7 @@ export default class WorkerManagerWorkerPoolsView extends Component {
           {WorkerManagerWorkerPoolSummaries && (
             <Fragment>
               <WorkerManagerWorkerPoolsTable
+                searchTerm={workerPoolSearch}
```

Comment:
Is `WorkerManagerWorkerPoolsTable` making use of this search term?

Reply:
Yes. `searchTerm` is passed to the `ConnectionDataTable` in `WorkerManagerWorkerPoolsTable` where it is rendered.